packages/syft/src/syft/service/settings/settings.py

```diff
@@ -98,8 +165,29 @@ def _repr_html_(self) -> Any:
                 <p><strong>Description: </strong>{self.description}</p>
                 <p><strong>Deployed on: </strong>{self.deployed_on}</p>
                 <p><strong>Signup enabled: </strong>{self.signup_enabled}</p>
-                <p><strong>Notifications enabled: </strong>{notifications_enabled}</p>
+                <p><strong>Notifications enabled: </strong>{notification_print_str}</p>
                 <p><strong>Admin email: </strong>{self.admin_email}</p>
             </div>
 
             """
+
+
+@migrate(ServerSettingsV1, ServerSettings)
+def migrate_server_settings_v1_to_v2() -> list[Callable]:
+    return []
```

Comment:
I think we should set a default value to `notifications_enabled` of `False` when upgrading the old ServerSettings version to new one, since `notifications_enabled` is a required field in the new version (and old version will not have it)

Reply:
Right, makes sense. Also, you meant the `ServerSettingsUpdate` class (and it's old version) as opposed to the `ServerSettings` class?

Reply:
No I meant `ServerSettings` class itself. Cause when we migrate old ServerSettings data to the new version, in that case, when we create the new ServerSettings class, we need to specify some value for the `notifications_enabled` attr, but since the old ServerSettings version doesn't have this attr defined, we will fail to migrate to the new ServerSetting class. So to fix this we need to define a migration that adds the `notification_enabled` attr with some default value to it.